java/pom.xml

```diff
@@ -15,7 +15,7 @@
         <dependency.lombok.version>1.18.30</dependency.lombok.version> <!-- 1.18.30 is min compatible with jdk 21+ -->
         <dependency.dagger.version>2.40.5</dependency.dagger.version>
         <dependency.jackson.version>2.15.2</dependency.jackson.version>
-        <dependency.apache-commons-lang3.version>3.12.0</dependency.apache-commons-lang3.version>
+        <dependency.apache-commons-lang3.version>3.13.0</dependency.apache-commons-lang3.version> <!-- July 2023 release, doesn't actually have constants for java after 17 yet -->
```

Comment:
it does, doesn't it?
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/JavaVersion.html#JAVA_21

Reply:
🤔 intellij must have had it cached for me, still didn't recognize it